### PR TITLE
fix(farms): Auction loading and auction only tag

### PR DESCRIPTION
**Pull request by chef-jojo**

### CODE REVIEW

File: src/views/Farms/components/FarmTable/Row.tsx [hunk 1]
```diff
@@ -151,11 +153,12 @@ const Row: React.FunctionComponent<RowPropsWithLoading> = (props) => {
           <FarmMobileCell colSpan={3}>
             <Flex justifyContent="space-between" alignItems="center">
               <Farm {...props.farm} />
-              {props.type === 'community' ? (
-                <FarmAuctionTag marginRight="16px" scale="sm" />
-              ) : (
-                <CoreTag marginRight="16px" scale="sm" />
-              )}
+              {farmAuctionLoaded &&
```
Comment (Chef-Yogi) [inline src/views/Farms/components/FarmTable/Row.tsx:156]: I can add skeleton here while loading after you merge this

**Comment by memoyil** [comment]
Currently fetchFarmAuction only sets isCommunity to true (not to false if not), that means after the auction period ends farm will not be visible as core instead of farmauction since you set the data statically. Is this intended or should we fix it?

**Comment by chef-jojo** [comment]
> Currently fetchFarmAuction only sets isCommunity to true (not to false if not), that means after the auction period ends farm will not be visible as core instead of farmauction since you set the data statically. Is this intended or should we fix it?

@memoyil farms eligible with farm auction only should stay as farm auction even after ends

**Comment by chefilip** [comment]
Screen Shot 2022-05-18 at 05 09 13

I think we need to `isCommunity` in this file as well @chef-jojo

**Comment by chef-jojo** [comment]
> Screen Shot 2022-05-18 at 05 09 13
> 
> I think we need to `isCommunity` in this file as well @chef-jojo

yeah we need this, it's not removed

**Pull request closed without merging**